### PR TITLE
[BitSail][LICENSE] update Copyright time to 2022-2023 in LICENSE Header File and Website

### LINKED ISSUE #351: [BitSail][LICENSE] update Copyright time to 2022-2023 in LICENSE Header File and Website
## Is your feature request related to a problem? Please describe

A clear and concise description of what the problem is. 

Example: I'm always frustrated when [...]

## Additional context
Copyright 2022-2023 Bytedance Ltd. and/or its affiliates.

### PR DESCRIPTION
**Pull request by wuchunfu**
Signed-off-by:

## Pre-Checklist

Note: Please complete **_ALL_** items in the following checklist.

- [x] I have read through the [CONTRIBUTING.md](https://github.com/bytedance/bitsail/blob/master/docs/contributing.md) documentation.
- [x] My code has the necessary comments and documentation (if needed).
- [x] I have added relevant tests.

## Purpose

Some description about what this PR wants to do.

## Approaches

Some description about how this PR achives the purpose. 

## Related Issues

close #351

## New Behavior (screenshots if needed)

N/A

### CODE REVIEW

File: .licenserc.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ header:
     spdx-id: Apache-2.0
     copyright-owner: ByteDance
     content: |
-      Copyright 2022 Bytedance Ltd. and/or its affiliates.
```
Comment (BlockLiu) [inline .licenserc.yaml:6]: Should we use `2022-2023`?

Comment (wuchunfu) [reply .licenserc.yaml:6]: @BlockLiu Thank you for your reminder. I will fix it later

Comment (wuchunfu) [reply .licenserc.yaml:6]: @BlockLiu done

**Comment by wuchunfu** [comment]
@BlockLiu @lichang-bd PTAL, thanks

**Comment by lichang-bd** [comment]
LGTM，thanks for you contribution @wuchunfu

**Pull request merged**